SPECS/tensorflow/tensorflow.spec

```diff
@@ -93,6 +95,40 @@ popd
 # Need to patch CVE-2024-35195 in the bundled python for applicable archs: `ExclusiveArch:  x86_64`
 pushd /root/.cache/bazel/_bazel_$USER/$MD5_HASH/external/
 patch -p1 < %{PATCH4}
+patch -p1 < %{PATCH5}
+
+pushd pypi__pip/
+patch -p1 < %{PATCH6}
+popd
+
+pushd python_aarch64-apple-darwin/lib/python3.12/site-packages/
```

Comment:
@KavyaSree2610 we generally only care about the aarch64 and x86_64 architectures for linux only.  This also appears to be changing an embedded version of python, which may not be necessary since we should be using the distribution version.

Reply:
@rikenm1 can you weigh in here?

Reply:
urllib3 package is a runtime requirement of Tensorflow. This can be disputed as we use system python3-urllib3 which is brought in by python3-request.

Reply:
python-zipp patch can stay.